src/v2/view-builder/views/DeviceChallengePollView.js

```diff
@@ -34,19 +33,34 @@ const Body = BaseForm.extend({
   },
 
   doChallenge () {
-    const deviceChallenge = this.options.appState.get('currentState')[
+    const deviceChallenge = this.options.appState.get(
       this.deviceChallengePollRemediation.relatesTo
-    ];
+    );
     switch (deviceChallenge.challengeMethod) {
     case 'LOOPBACK':
+      this.title = loc('signin', 'login');
+      this.add('<div class="spinner"></div>');
       this.doLoopback(deviceChallenge.domain, deviceChallenge.ports, deviceChallenge.challengeRequest);
       break;
     case 'CUSTOM_URI':
-      this.doCustomURI(deviceChallenge.href);
+      this.title = 'Verify account access';
+      this.subtitle = 'Launching Okta Verify...';
+      this.add(`
+        If nothing prompts from the browser, 
+        <a href="#" id="launch-ov" class="link">click here</a> to launch Okta Verify, 
+        or <a href="#" id="download-ov" class="link">download and run Okta Verify</a>.
```

Comment:
is there a plan to implement the `download-ov` link?

Reply:
We will not have the implementation for BETA. We will document this so the customers do not expect the download link to be working.

Reply:
Would be it slightly better to remove the content from the page at all?

Reply:
I don't have strong opinion, but based on the slack channel discussion as I remember, this is the text the team agreed on.

Reply:
I second that, maybe we should think about removing the `download-ov` link. Otherwise this is this just going to be a link without any action right ?

Reply:
@nbories-okta ^ would you like to reword the sentence so that we don't mention of downloading the OV?